docs/conductor/getting-started.md

```diff
@@ -0,0 +1,93 @@
+# Getting started with Conductor
+##
+
+Conductor will group and schedule automated Dependency Updates on your own Continuous Integration platform. If the update succeeds, Conductor sends you a pull request to your code hosting platform (GitHub, GitLab, Bitbucket) with the changed composer.lock file and, if necessary, other files modified by Composer plugins or scripts.
+
+To use Conductor:
+
+- You need to be approved for early access to conductor. [Join to the waitlist](http://packagist.com.lo/features/conductor) and wait for approval.
+- You need a Private Packagist trial or subscription (Cloud or Self-Hosted).
+- You need to add a workflow to your Continuous Integration platform to run Composer updates, described below.
+
+## First steps
+
+Once you have a Private Packagist Subscription and Conductor is enabled for your subscription, log into your Private Packagist organization and click on the "Updates" tab in the main navigation.  
+Conductor will list packages:
+ - added to Private Packagist via synchronization
+ - having a composer.lock file commited to the repository
+ - having the Private Packagist repository added to the composer.json
+
+If you do not see your package, follow the instructions on your organization page to add the custom repository to the composer.json of your package.  
+
+Most convenient way to set up Conductor is to configure your CI in the Private Packagist UI. The manual steps are outlined below:
```

Comment:
```suggestion
The most convenient way to set up Conductor is to configure your CI in the Private Packagist UI. The manual steps are outlined below:
```
This sounds a bit like Private Packagist provides functionality to configure the CI directly. All we do is list steps that they need to follow to set up CI.